api/transactions.js

```diff
@@ -224,14 +228,15 @@ function setTransactionBitFlags(transaction, options) {
  *
  * @param {RippleAddress} account
  * @param {Hex-encoded String|ASCII printable character String} identifier
+ * @param {Object} options
  * @param {Function} callback
  *
  * @callback
  * @param {Error} error
  * @param {Transaction} transaction
  */
-function getTransaction(api, account, identifier, callback) {
-  var options = {};
+function getTransaction(api, account, identifier, options, callback) {
+  assert.strictEqual(typeof options, 'object');
```

Comment:
Actually, its everywhere. Maybe keep the `var options = {}` at the top of the outer function, but call it something else, so we don't worry about mutating the `options` being passed into the function